pkg/minikube/driver/driver.go

```diff
@@ -141,17 +145,69 @@ func IsDocker(name string) bool {
 	return name == Docker
 }
 
-// IsDockerDesktop checks if the driver is a Docker for Desktop (Docker on windows or MacOs)
+func IsLinuxDockerDesktop(name string) bool {
+	if IsDocker(name) {
+		cmd := exec.Command("docker", "version")
+		stdout, err := cmd.StdoutPipe()
+		if err != nil {
+			return false
+		}
+		if err := cmd.Start(); err != nil {
+			return false
+		}
+		data, err := ioutil.ReadAll(stdout)
+		if err != nil {
+			return false
+		}
+		output := string(data)
```

Comment:
This can be written in a shorter way using `Output()`.
```
data, err := exec.Command("docker", "version").Output()
if err != nil {
    return false
}
output := string(data)
```